src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component } from 'react';
+import './Counter.scss';
+
+type State = {
+  count: number,
+};
+
+export class Counter extends Component<{}, State> {
+  state = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((state) => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((state) => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0 && this.state.count > 0) {
+      this.addOne();
+      this.add100();
+    } else {
+      this.addOne();
+    }
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>
+          Count:
+          {` ${this.state.count}`}
+        </h1>
+        <button
```

Comment:
add 1 empty line between same level elements
```suggestion
        </h1>
        
        <button
```